packages/ibm-products/src/components/PageHeader/PageHeader.js

```diff
@@ -528,6 +528,7 @@ export let PageHeader = React.forwardRef(
                               ? [breadcrumbItemForTitle]
                               : breadcrumbs // breadcrumbs may be null or undefined
                           }
+                          overflowTooltipAlign="right"
```

Comment:
Probably better not to hardcode this. While currently this may be the only one that fully displays at this moment, there may be cases in the future where this is addressed and teams are able to set a different position.

Reply:
`overflowTooltipAlign={breadcrumbOverflowTooltipAlign}`